specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-11-01/consumption.json

```diff
@@ -0,0 +1,5397 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "usageEnd": {
+          "description": "Usage end date.",
+          "type": "string",
+          "readOnly": true
+        },
+        "azureCharges": {
+          "description": "Azure Charges.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        },
+        "chargesBilledSeparately": {
+          "description": "Charges Billed separately.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        },
+        "marketplaceCharges": {
+          "description": "Marketplace Charges.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        },
+        "billingAccountId": {
+          "description": "Billing Account Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "billingProfileId": {
+          "description": "Billing Profile Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionId": {
+          "description": "Invoice Section Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerId": {
+          "description": "Customer Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "isInvoiced": {
+          "description": "Is charge Invoiced",
+          "type": "boolean",
+          "readOnly": true
+        }
+      }
+    },
+    "MeterDetailsResponse": {
+      "description": "The properties of the meter detail.",
+      "properties": {
+        "meterName": {
+          "description": "The name of the meter, within the given meter category",
+          "type": "string",
+          "readOnly": true
+        },
+        "meterCategory": {
+          "description": "The category of the meter, for example, 'Cloud services', 'Networking', etc..",
+          "type": "string",
+          "readOnly": true
+        },
+        "meterSubCategory": {
+          "description": "The subcategory of the meter, for example, 'A6 Cloud services', 'ExpressRoute (IXP)', etc..",
+          "type": "string",
+          "readOnly": true
+        },
+        "unitOfMeasure": {
+          "description": "The unit in which the meter consumption is charged, for example, 'Hours', 'GB', etc.",
+          "type": "string",
+          "readOnly": true
+        },
+        "serviceFamily": {
+          "description": "The service family.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorResponse": {
```

Comment:
Please reference the default error response: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json#L277  unless this results in a breaking change

Reply:
Please indicate whether this would be a breaking change.  There is good reason to use the common definition - it allows uniform handling of errors across RPs fro generated SDKs (and also for customers using the REST API).

Reply:
yes, it ends up with breaking changes.